### PR TITLE
check cost model values in dbsync

### PR DESCRIPTION
**Pull request by saratomaz**

### CODE REVIEW

File: cardano_node_tests/utils/dbsync_queries.py [hunk 1]
```diff
@@ -691,3 +691,13 @@ def query_datum(datum_hash: str) -> Generator[DatumDBRow, None, None]:
     with execute(query=query, vars=(rf"\x{datum_hash}",)) as cur:
         while (result := cur.fetchone()) is not None:
             yield DatumDBRow(*result)
+
+
+def query_cost_model() -> Any:
```
Comment (mkoura) [inline cardano_node_tests/utils/dbsync_queries.py:696]: should be probably `-> Dict[str, Any]`

File: cardano_node_tests/tests/test_dbsync.py [hunk 2]
```diff
@@ -208,3 +208,25 @@ def test_blocks(self, cluster: clusterlib.ClusterLib):  # noqa: C901
...
+
+        pp_cost_models = protocol_params["costModels"]
+        db_cost_models = dbsync_queries.query_cost_model()
+
+        assert set(pp_cost_models["PlutusScriptV1"]) == set(
+            db_cost_models["PlutusV1"]
+        ), "PlutusV1 cost model is not the expected"
+        assert set(pp_cost_models["PlutusScriptV2"]) == set(
+            db_cost_models["PlutusV2"]
+        ), "PlutusV2 cost model is not the expected"
```
Comment (mkoura) [inline cardano_node_tests/tests/test_dbsync.py:232]: Why don't we compare whole models including values, and not just key names?

**Review by mkoura — CHANGES_REQUESTED** [review]

File: cardano_node_tests/utils/dbsync_queries.py (hunk 1 above)
Comment (saratomaz) [reply cardano_node_tests/utils/dbsync_queries.py:696]: mypy doesn't allow it

Comment (mkoura) [reply cardano_node_tests/utils/dbsync_queries.py]: See https://github.com/input-output-hk/cardano-node-tests/pull/1563/commits/ac0841022a0b9bddb61e94b14cb9cd96289c170c

**Review by mkoura — APPROVED** [review]

**Pull request merged**